str/runners/str_iterative_eh_runner.py

```diff
@@ -29,21 +31,26 @@
 
 
 # inputs:
-# variant catalog
-@click.option('--variant-catalog', help='Full path to Illumina Variants catalog')
-# input dataset
+@click.option(
+    '--variant-catalog',
+    help='Full path to Illumina Variants catalog (sharded/unsharded)',
+)
 @click.option('--dataset', help='dataset eg tob-wgs')
-# input sample ID
-@click.argument('input_cpg_sids', nargs=-1)
 @click.option(
     '--max-parallel-jobs',
     type=int,
     default=50,
     help=('To avoid exceeding Google Cloud quotas, set this concurrency as a limit.'),
 )
+# sample id and sex mapping file
+@click.option('--sample-id-file', help='Full path to mapping of CPG id and sex')
+@click.option(
+    '--job-storage', help='Storage of the Hail batch job eg 30G', default='70G'
+)
+@click.option('--job-memory', help='Memory of the Hail batch job', default='highmem')
 @click.command()
 def main(
-    variant_catalog, dataset, max_parallel_jobs, input_cpg_sids: list[str]
+    variant_catalog, dataset, max_parallel_jobs, sample_id_file, job_storage, job_memory
```

Comment:
can you add some type hinting here

```suggestion
    variant_catalog: str, dataset: str, max_parallel_jobs: int, sample_id_file: str, job_storage: str, job_memory: str
```